### ISSUE TITLE
Do not use script-wide use strict statement

### ISSUE THREAD

**Issue by dekr** [kenwheeler/cash#49]
You are using a script-wide form of use strict statement. This may cause problems with minimizing. You can use a function form instead:

```
(function(document, window, ...) {
    "use strict";
})(document, window, ...);
```

There is more explanation on the following stackoverflow-thread: [stackoverflow link](http://stackoverflow.com/questions/4462478/jslint-is-suddenly-reporting-use-the-function-form-of-use-strict)

**Comment by kenwheeler** [comment]
thanks, looking into this

**Comment by shshaw** [comment]
That seems to be automatically added by one of the preprocessors as it isn't present in the actual _wrapper.js file. I am trying to track down which one is doing it, and I think 6to5 is likely the culprit. Since 6to5 is now Babel, it may be worthwhile to migrate over to Babel to get the latest updates.

**Comment by thasmo** [comment]
@shshaw This is somewhat of a big issue. I've problems with including different libraries when they're included after `Cash`. If I include those libraries before `Cash` everything works. Wondering that not more people have found this issue. For now I can only include `Cash` as the last script.

**Comment by shshaw** [comment]
Yeah, I want to fix this as well. 

It's tied into #93 ( and #90 / #92 ), which is a big change to the build process. @simeydotme was helping with this, but I'm not sure if he's made any more progress. If you have Babel experience or can help with the build process, then please contribute.

**Comment by thasmo** [comment]
Unfortunately I don't have experience with Babel nor 6to5.

**Comment by shshaw** [comment]
Unfortunately I don't either 😄  I've stumbled around with it a bit, but their documentation is quite cryptic, so I haven't wanted to deploy anything without someone experienced taking a look.

**Comment by shshaw** [comment]
Took another stab at this, and managed a gulp-free build process using babel and uglify.

Pushed to the [build-revision](https://github.com/kenwheeler/cash/tree/build-revision) branch.

**Comment by rcoelho** [comment]
Works for me!
Thanks for preparing this build, it took me some time to land here; when I did, problem solved :)

**Comment by fabiospampinato** [comment]
This will be addressed in the next milestone.

**Comment by fabiospampinato** [comment]
Fixed in v2.1.3.

It was surprisingly tricky to implement, the minifiers kind of got in the way.
